scripts/generate-data.js

```diff
@@ -20,60 +16,14 @@ const getTime = () => {
 		pad(currentdate.getSeconds());
 };
 
-const complicatedWorkThatTakesTime = (resource, callback) => {
-
-	if (resource.length) {
-
-		const version = resource[0].version;
-		console.log('[%s] Worker %d \u2192 Unicode v%s',
-			getTime(), cluster.worker.id, version);
-
+const main = () => {
+	for (const resource of resources) {
+		const version = resource.version;
+		console.log('[%s] Unicode v%s', getTime(), version);
+		console.groupCollapsed();
```

Comment:
As a bonus, this refactor also fixes another build hazard: Previously when there is an error thrown from the building worker, the cluster main does not exit early, nor does it exit with non-zero code. Now that all building steps are within one process, any thrown errors should invalidate the build step.

Reply:
Fixed the mentioned build hazard in https://github.com/node-unicode/node-unicode-data/pull/83/commits/c6f9130c3e4b29f668094d7567e7a9d137c372a3.

See https://github.com/node-unicode/node-unicode-data/actions/runs/11724848834/job/32659747285 for the error log example.